pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -115,11 +115,8 @@ func setCABundle(caBundle []byte) error {
 	if ok := rootCAs.AppendCertsFromPEM(caBundle); !ok {
 		return errors.New("no certs were appended from caBundle")
 	}
-	client.Transport = &http.Transport{
-		TLSClientConfig: &tls.Config{
-			RootCAs: rootCAs,
-		},
-	}
+	TLSClientConfig := &tls.Config{}
```

Comment:
What we need to do is update this `client` global variable here:
https://github.com/googleforgames/agones/blob/0ec68ff6953e30868615f3ca333876bce7f3b894/pkg/fleetautoscalers/fleetautoscalers.go#L38

What is happening here is you are changing a local variable now, which isn't doing anything specific.